src/formatDate.js

```diff
@@ -50,7 +50,26 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat.pop();
```

Comment:
The use of `pop` on `fromFormat` mutates the original array, which is generally not a good practice as it can lead to unexpected side effects. Consider using a different approach that does not modify the input array.